lib/mongo/crypt/handle.rb

```diff
@@ -52,6 +52,31 @@ def ref
 
       private
 
+      # Send the logs from libmongocrypt to the Mongo::Logger
+      def set_logger
+        @log_callback = Proc.new do |level, msg|
+          logger = Mongo::Logger.logger
```

Comment:
This takes the global logger, not the client's logger for any particular client. This is an existing issue in the driver affecting a number of call sites, but is the client's logger conveniently available here? Or, is the logger here intended to be the global one?

Reply:
Nope, you're right, I meant it to be the client's logger. I'll fix it. Thanks for pointing that out.